kotlinpoet/src/main/java/com/squareup/kotlinpoet/ParameterSpec.kt

```diff
@@ -183,9 +183,13 @@ class ParameterSpec private constructor(
 internal fun List<ParameterSpec>.emit(
   codeWriter: CodeWriter,
   forceNewLines: Boolean = false,
+  forceParensOnEmpty: Boolean = true,
```

Comment:
I'd really prefer these changes to be in a separate PR, looks they're not strictly related to this one. Can you please pull them out and rebase this PR on top? Should be an easy merge.

Reply:
https://github.com/square/kotlinpoet/pull/774